test/fixtures/jsx/basic/entity/expected.json

```diff
@@ -136,12 +136,12 @@
                 }
               },
               "extra": null,
-              "value": "💩"
+              "value": "\uD83D\uDCA9"
```

Comment:
Note: this doesn’t change the actual string value (`"\uD83D\uDCA9" === "💩"`), it just ensures the file is ASCII-safe.